Dockerfiles/cluster-agent/entrypoint.sh

```diff
@@ -13,6 +13,9 @@ if [[ -z "$DD_API_KEY" ]]; then
     exit 1
 fi
 
+##### Copy the custom confs #####
+find /conf.d -name '*.yaml' -exec cp --parents -fv {} /etc/datadog-agent/ \;
```

Comment:
Can't we target a specific folder for custom config files ? It feels like we might end up copying unexpected files to the datadog-agent folder.

Reply:
This logic is copied from the agent's container, and it handles copying sub-folders. It means:
  - copy every file found in `/conf.d` to `/etc/datadog-agent/conf.d/`
  - copy every file found in `/conf.d/mysql.d` to the new `/etc/datadog-agent/conf.d/mysql.d/` folder